browser-test/src/support/admin_predicates.ts

```diff
@@ -17,7 +17,10 @@ export class AdminPredicates {
     operator: string,
     value: string,
   ) {
-    await this.page.click(`text="${questionName}"`)
+    const addQuestionButtonLocator = this.page.locator('button', {
+      has: this.page.locator(`:text("Admin ID: ${questionName}")`),
```

Comment:
Is it different from `this.page.locator('button:has-text("Admin ID...")')`?

Reply:
Good call. Done.